### DISCUSSION TITLE
支持卷快照功能吗

### DISCUSSION THREAD

**Discussion by caijinhu123** [juicedata/juicefs#4153] [Q&A]
我使用csi创建了卷快照类
`apiVersion: snapshot.storage.k8s.io/v1beta1
deletionPolicy: Delete
driver: csi.juicefs.com
kind: VolumeSnapshotClass
metadata:
  generation: 1
  managedFields:
    - apiVersion: snapshot.storage.k8s.io/v1
      fieldsType: FieldsV1
      fieldsV1:
        'f:deletionPolicy': {}
        'f:driver': {}
      manager: controller-manager
  name: juicefs-sc`
再基于该快照类创建了卷快照，但是卷快照一直提示创建中
`apiVersion: snapshot.storage.k8s.io/v1
kind: VolumeSnapshot
metadata:
  name: pvc-snap-cc # 替换为你的快照名称
spec:
  volumeSnapshotClassName: juicefs-sc
  source:
    persistentVolumeClaimName: juicefs-pvc`
查看卷快照提示：Waiting for a snapshot default/pvc-snap-cc to be created by the CSI driver.

**Comment by zwwhdls** [comment]
目前还不支持

**Comment by caijinhu123** [comment]
收到，谢谢
